### DISCUSSION TITLE
Pipenv won't work in the codepsace

### DISCUSSION THREAD

**Discussion by Dhruvacube** [community/community#39056] [Codespaces]
### Select Topic Area

Bug

### Body

Pipenv won't work properply, now here i am clueless on how to enable github in codespaces

![image](https://user-images.githubusercontent.com/34207400/201635458-6ecbde58-3e39-4c06-a32c-fb73aefbd4b5.png)

This is my repo https://github.com/The-4th-Hokage/yondaime-hokage

**Comment by erikaellr** [comment]
Hi @Dhruvacube,

Thanks for being a part of the GitHub Community! We're glad you're here.

This doesn't look like an issue specific to Codespaces - the line "ERROR: cannot install discord-py 2.1.0 ... and discord-py 2.2.0 ... because these package versions have conflicting dependencies" means that there is a conflict in your dependency graph that pipenv cannot resolve. I believe that the pipenv docs suggest `pipenv graph` for inspecting the dependency graph and figuring out which packages are resulting in such a conflict so that you can fix it, but the best source of help on that will be the [pipenv docs](https://pipenv.pypa.io/en/latest/) or other resources or forums specific to pipenv.

Best of luck!
